frontend/ui/widget/touchmenu.lua

```diff
@@ -162,12 +162,13 @@ function TouchMenuBar:init()
     local icon_sep_width = Size.span.vertical_default
     local icons_sep_width = icon_sep_width * (#self.icons + 1)
     -- we assume all icons are of the same width
-    local tmp_ib = IconButton:new{icon_file = self.icons[1]}
+    local icon_width = Screen:scaleBySize(40)
+    local icon_height = icon_width
     -- content_width is the width of all the icon images
-    local content_width = tmp_ib:getSize().w * #self.icons + icons_sep_width
+    local content_width = icon_width * #self.icons + icons_sep_width
     local spacing_width = (self.width - content_width)/(#self.icons*2)
-    local icon_padding = math.min(spacing_width, Screen:scaleBySize(20))
-    self.height = tmp_ib:getSize().h + Screen:scaleBySize(10)
+    local icon_padding = math.min(spacing_width, Size.span.horizontal_default)
```

Comment:
Would look fine (= as before) to me on kobo with a `Size.span.horizontal_menuicons = scaleBySize(16)` (instead of `horizontal_default = Screen:scaleBySize(10)`)

Reply:
I thought 20 looked on the large side but I suppose half that might indeed be a bit small.

Should it be in Size? This seems like pretty much a one-off value to me. For the bottom menu perhaps?

Reply:
Dunno, depend if that same value plays well in the bottom menu too. Same for your `icon_width = Screen:scaleBySize(40)`.
And if it happens it works as well on the InfoMessage icon and others alike, it may well fit in Size.
May be with a specific key (but it mess with your existing hierarchy padding > icons):
```lua
Size.icons = {
 horizontal_pading =  Screen:scaleBySize(16),
 width = Screen:scaleBySize(40),
 ...
}
```